daprdocs/content/en/reference/components-reference/supported-bindings/commercetools.md

```diff
@@ -0,0 +1,71 @@
+---
+type: docs
+title: "commercetools GraphQL binding spec"
+linkTitle: "commercetools GraphQL"
+description: "Detailed documentation on the commercetools GraphQL binding component"
+aliases:
+  - "/operations/components/setup-bindings/supported-bindings/commercetools/"
+---
+
+## Component format
+
+To setup commercetools GraphQL binding create a component of type `bindings.commercetools`. See [this guide]({{< ref "howto-bindings.md#1-create-a-binding" >}}) on how to create and apply a binding configuration.
+
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: bindings.commercetools
+  version: v1
+  metadata:
+  - name: Region # required.
```

Comment:
Comparing to your code PR all these meatadata fields must start with a lowercase character

Reply:
ping @harrykimpel

Reply:
thanks @yaron2, will update

Reply:
done